src/allmydata/immutable/layout.py

```diff
@@ -254,7 +254,7 @@ def close(self):
         return d
 
     def abort(self):
-        self._rref.callRemote("abort").addErrback(log.err, "Error from remote call to abort an immutable write bucket")
+        return self._rref.callRemote("abort").addErrback(log.err, "Error from remote call to abort an immutable write bucket")
```

Comment:
Hm, callers of this one are trickier to track down.  Five different `abort` definitions in Tahoe...

But the only caller of an `abort` method I can find that does anything with the return value seems to be calling `allmydata.immutable.encoder.Encoder.abort`, not this one.